js/pages/statisticsPage/statisticFilter.js

```diff
@@ -0,0 +1,158 @@
+import { createRow } from '../pagesLayout/statisticLayout';
+
+function cleanTBody() {
+  const tbody = document.querySelector('tbody');
+  while (tbody.firstChild) {
+    tbody.firstChild.remove();
+  }
+}
+
+function updateStatisticContent(num, category, word, translation, trainClick, playClick, errors) {
+  const tbody = document.querySelector('tbody');
+  let percent = Math.floor((errors / playClick) * 100);
+  if (!percent) {
+    percent = 0;
+  }
+  const row = document.createElement('tr');
+  createRow(row, num);
+  createRow(row, (category.charAt(0).toUpperCase() + category.slice(1)));
+  createRow(row, word);
+  createRow(row, translation);
+  createRow(row, trainClick, 'train-column');
+  createRow(row, playClick, 'play-column');
+  createRow(row, errors, 'errors-column');
+  createRow(row, percent, 'percent-column');
+  tbody.appendChild(row);
+}
+
+function returnAllWords() {
+  const statistic = JSON.parse(localStorage.getItem('statistic'));
+  const keys = Object.keys(statistic);
+
+  const arr = [];
+  keys.forEach((key) => {
+    if (key !== 'sections') {
```

Comment:
I saw the 'sections' in a couple of files, is it worth to move it in global constant?